uninstall.adoc

```diff
@@ -5,11 +5,16 @@ When you uninstall
 identity configuration management for Kubernetes, it might take up to 5 minutes to complete the uninstall process.
 The uninstall removes most operator components, excluding components such as custom resource definitions. After this uninstall, you must reinstall the operator before reinstalling the custom resource.
 
-[#prerequisite-deletee-authrealm]
-== Prerequisite: Delete AuthRealm custom resources
+[#prerequisite-delete-idpconfig-authrealm]
+== Prerequisite: Delete the IDPConfig and AuthRealm custom resources
 
 Before you uninstall the 
-identity configuration management for Kubernetes, you must delete all of the AuthRealm custom resources that are managed by the operator.
+identity configuration management for Kubernetes: 
+
+. You must delete all of the AuthRealm custom resources that are managed by the operator.
+
+. You must delete the IDPConfig CR, this will delete the different operators except the installer. The installed will be deleted when you delete the identity configuration management from the OperatorHub.
```

Comment:
Propose: 
You must delete the IDPConfig CR, this will delete several operators except the installer operator. The installer operator will be deleted when you delete the identity configuration management from the hub